// Solution-2.scala
// Solution to Exercise 2 in "Error Handling with Exceptions"
import com.atomicscala.AtomicTest._

/*{oldDescription}
2.	Create a class with a simple method f. Create a var of that class and initialize it to null. Try to call f using this var. Now wrap the call in a try-catch clause to catch the exception.
{oldDescription}*/

// needs solution
